### DISCUSSION TITLE
Issue related to pushing my local repository to git hub repository

### DISCUSSION THREAD

**Discussion by SuriaPrakash1998** [community/community#63883] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was using a old account of my which was suspended. I dont know  the reason. I have to submit my project immediately hence i opened a new account . But i find the error below while pushing  my coded . I tried for manual upload, which  i could do. 
but i cant push my code through terminal. 
remote: Your account is suspended. Please visit https://support.github.com for more information.
fatal: unable to access 'https://github.com/SuriaPrakash1998/shopify.git/': The requested URL returned error: 403

**Comment by melvincwng** [comment]
Hey @SuriaPrakash1998, can you try the following command in your terminal `git config -l | grep user.name`

This command would show you which GitHub account you're currently signed into in your terminal.

It could be because you're still signed in as your old GitHub account in the terminal (which was suspended), hence it's returning error 403 and forbidding you to push your code.

If that's the case, you need to sign in to your new GitHub account in the terminal and try pushing your code into that mentioned repository again.

Hope this helps and answers your question 😄!

Potentially useful references to look at:
- [How to change git account in git bash](https://stackoverflow.com/questions/41689395/how-to-change-git-account-in-git-bash)
- [How to sign out in Git bash console in Windows](https://stackoverflow.com/questions/38549834/how-do-i-sign-out-in-the-git-bash-console-in-windows/38553149#38553149)

**Reply by Virginians** [reply]
Correlation Id:
